projects/sm-pwr/copy.ipynb

```diff
@@ -0,0 +1,1543 @@
+{
```

Comment:
<p>Delete this copy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dpploy/engy-4390/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>